src/test/java/de/mediathekview/mserver/crawler/zdf/json/ZdfFilmDetailDeserializerTest.java

```diff
@@ -0,0 +1,98 @@
+            "https://www.zdf.de/filme/das-duo/das-duo-echte-kerle-102.html"
+        },
+        {
+            "/zdf/zdf_film_details2.json",
+            "logo!",
+            "logo! am Freitagabend",
+            LocalDateTime.of(2018, 2, 23, 19, 25, 0),
+            Duration.ofMinutes(8).plusSeconds(14),
+            "Schaut euch hier die logo!-Sendung von Freitagabend noch einmal an! Die Sendungen sind eine Woche online.",
+            "https://www.zdf.de/kinder/logo/logo-am-freitagabend-104.html"
+        },
+        {
+            "/zdf/zdf_film_details3.json",
+            "JoNaLu",
+            "Tanz auf dem Seil - Folge 25",
+            LocalDateTime.of(2018, 3, 11, 9, 50, 0),
+            Duration.ofMinutes(24).plusSeconds(55),
+            "Naya verliert beim Seiltanz ihre Glücksblume und alles geht schief. Kann ein anderer Glücksbringer helfen? Glühwürmchen Minou hat eine \"leuchtende\" Idee.",
+            "https://www.zdf.de/kinder/jonalu/tanz-auf-dem-seil-102.html"
+        },
+        {
+            "/zdf/zdf_film_details4.json",
+            "ZDF-Morgenmagazin",
+            "Oscar-Verleihung in Los Angeles",
+            LocalDateTime.of(2018, 3, 2, 5, 30, 0),
+            Duration.ofMinutes(2).plusSeconds(34),
+            "Die diesjährigen Oscar-Gewinner stehen schon fest. Denn die rund 8000 Juroren haben ihre Stimmzettel längst eingereicht. Am Sonntag werden die Oscars zum 90. Mal vergeben - und dann kennen auch wir die Gewinner.",
+            "https://www.zdf.de/nachrichten/zdf-morgenmagazin/videos/oscars-106.html"
+        }
+    });
+  }
+
+  private final String jsonFile;
+  private final String expectedTopic;
+  private final String expectedTitle;
+  private final LocalDateTime expectedTime;
+  private final Duration expectedDuration;
+  private final String expectedDescription;
+  private final String expectedWebsite;
+
+  public ZdfFilmDetailDeserializerTest(final String aJsonFile, final String aExpectedTopic, final String aExpectedTitle,
+      final LocalDateTime aExpectedTime, final Duration aExpectedDuration, final String aExpectedDescription,
+      final String aExpectedWebsite) {
+    jsonFile = aJsonFile;
+    expectedTopic = aExpectedTopic;
+    expectedTitle = aExpectedTitle;
+    expectedTime = aExpectedTime;
+    expectedDuration = aExpectedDuration;
+    expectedDescription = aExpectedDescription;
+    expectedWebsite = aExpectedWebsite;
+  }
+
+  @Test
+  public void test() {
+    final JsonObject json = JsonFileReader.readJson(jsonFile);
+    final ZdfFilmDetailDeserializer target = new ZdfFilmDetailDeserializer();
+
+    final Optional<Film> actual = target.deserialize(json, Film.class, null);
+
+    assertThat(actual.isPresent(), equalTo(true));
+
+    AssertFilm.assertEquals(actual.get(), Sender.ZDF, expectedTopic, expectedTitle, expectedTime, expectedDuration, expectedDescription,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "actual.isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3655)